code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -448,4 +448,19 @@
 			node.addMember(src)
 		build_network()
 
+/obj/machinery/atmospherics/components/unary/cryo_cell/CtrlClick(mob/user)
+	if(on)
+		on = FALSE
+		update_icon()
```

Comment:
```suggestion
```

Reply:
> 

Can you explain why this isnt needed, I needed to add it in testing.

Reply:
> > 
> 
> Can you explain why this isnt needed, I needed to add it in testing.

Nevermind im a brick